proposals/dao/tip_121c_pt2.ts

```diff
@@ -0,0 +1,113 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  PcvStats,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+
+/*
+
+Tribe Redemption 
+
+*/
+
+// Circulating amount of TRIBE, which is redeemable for underlying PCV assets
+// TODO: Update with final circulating TRIBE figure
+const REDEEM_BASE = ethers.constants.WeiPerEther.mul(458_964_340);
+
+// Lido deposit balance, being withdrawn and sent to Tribe Redeemer
+const STETH_DEPOSIT_BALANCE = '50296523674661485703301';
```

Comment:
this should be a ratioPCVController transfer given it rebases upward over time

Reply:
This is used for a greaterThan test in validate() step, the proposal actually moves 100% with ratioPCVController 👍

Reply:
Yep as Erwan said